tyk-headless/values.yaml

```diff
@@ -14,11 +14,16 @@ redis:
     addrs:
       - redis.tyk.svc.cluster.local:6379
     useSSL: false
-    #If you're using Bitnami Redis chart please input the correct host to your installation in the field below
-    #addrs:
-      #- tyk-redis-master.tyk.svc.cluster.local:6379
-    #If you're using Bitnami Redis chart please input your password in the field below
-      #pass: ""
+#    #If you're using Bitnami Redis chart please input the correct host to your installation in the field below
```

Comment:
I think it'd better to address other managed redis services here too rather than redis bitnami chart.

Reply:
@abhishiekgowtham Can you elaborate a bit more on that? The only managed redis service we recommend to our user based on the docs at the moment is from Bitnami, not sure what other managed redis services you're referring to here?

Reply:
Cool, got it now and what you're saying make sense. However this change is out of scope of this PR and I believe it will require a bit more work than just changing a comment inside the values file eg. proper testing of those options you're suggesting, updating the docs with some examples etc.

Reply:
Sure, if you feel like it's out of the scope for this one, we can tackle that in a different PR.